pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -470,6 +468,7 @@ stdenv.mkDerivation (finalAttrs: {
     (enableFeature withMysofa "libmysofa")
     (enableFeature withOpus "libopus")
     (optionalString (versionAtLeast version "5.0" && withLibplacebo) "--enable-libplacebo")
+    (optionalString (versionAtLeast version "6.1" && withAribcaption) "--enable-libaribcaption")
```

Comment:
this version check is probably redundant now

Reply:
Removed it.